tests/test_apply.py

```diff
@@ -67,23 +68,23 @@ def test_apply_operation_single_wire_no_parameters(
         assert np.allclose(dev.state, np.array(expected_output), atol=tol, rtol=0)
         assert dev.state.dtype == dev.C_DTYPE
 
-    @pytest.mark.skipif(
-        device_name == "lightning.kokkos" or device_name == "lightning.gpu",
-        reason="Only meaningful for lightning_qubit",
-    )
-    @pytest.mark.skipif(not ld._CPP_BINARY_AVAILABLE, reason="Lightning binary required")
-    @pytest.mark.parametrize("operation,input,expected_output", test_data_no_parameters)
-    @pytest.mark.parametrize("C", [np.complex64, np.complex128])
-    def test_apply_operation_preserve_pointer_single_wire_no_parameters(
-        self, qubit_device, operation, input, expected_output, C
-    ):
-        dev = qubit_device(wires=1)
-        dev._state = dev._asarray(input, dtype=C)
-        pointer_before, _ = dev._state.__array_interface__["data"]
-        dev.apply([operation(wires=[0])])
-        pointer_after, _ = dev._state.__array_interface__["data"]
-
-        assert pointer_before == pointer_after
+    # @pytest.mark.skipif(
```

Comment:
I think these tests don't make sense anymore, I intend to remove it.
Let me know if otherwise.